cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -180,8 +180,11 @@ class parquet_metadata {
  public:
   /// Key-value metadata in the file footer.
   using key_value_metadata = std::unordered_map<std::string, std::string>;
-  /// row group metadata from each RowGroup element.
+  /// Row group metadata from each RowGroup element.
   using row_group_metadata = std::unordered_map<std::string, int64_t>;
+  /// Column chunk metadata from each ColumnChunkMetaData element.
+  using column_chunk_metadata =
+    std::unordered_map<std::string, std::vector<std::unordered_map<std::string, int64_t>>>;
```

Comment:
Right now we're exposing the metadata like
```
Map[Str, List[Map[Str, Int]]]
```
Ie. For each column name, we have a list of per-row-group metadata maps. The inner maps have the uncompressed and compressed sizes. Right now, we only need the uncompressed size in cudf-polars . 

I think we should simplify to `Map[Str, List[Int]]` and just expose the uncompressed size. We can always extend later if necessary.

Reply:
Ok of course.

Reply:
All done!